### PR TITLE
Transactions benchmarks

### PR DESCRIPTION
**Pull request by seglo**
Benchmarks for reactive-kafka transactional support.  I also included a `docker-compose.yml` file with up-to-date community Apache Kafka docker images that can be used with the tests.

### CODE REVIEW

**Comment by seglo** [comment]
Resolves issue #461

**Comment by seglo** [comment]
There's a failure happening in `ReconnectSpec`, but it doesn't look related.

https://travis-ci.org/akka/reactive-kafka/jobs/384851107#L1312

FYI: @2m, @ennru

File: benchmarks/docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    image: wurstmeister/zookeeper:3.4.6
+    ports:
+      - "2181:2181"
+  kafka:
+    image: wurstmeister/kafka:1.1.0
+    ports:
+      - "9092:9092"
+    environment:
+      # On Mac you may need to change `localhost` in KAFKA_ADVERTISED_LISTENERS to the ip/host of your docker machine ip
+      KAFKA_ADVERTISED_LISTENERS: INSIDE://localhost:9092,OUTSIDE://localhost:9094
```
Comment (2m) [inline benchmarks/docker/docker-compose.yml]: Nice. I was playing around with Kafka in docker containers as well and noticed that the `KAFKA_ADVERTISED_HOST_NAME` is deprecated.

**Comment by seglo** [comment]
Thanks.  Yeah, I think running the benchmarks will also reveal more instances of transient failures we've been seeing (I just found another one I'll log an issue for).  

BTW, I added another commit for some comments cleanup I did this morning: #483.

**Pull request merged**